src/openfl/_internal/stage3D/opengl/GLCubeTexture.hx

```diff
@@ -34,16 +34,19 @@ class GLCubeTexture {
 
 		var hasTexture = false;
 
-		reader.readTextures(function(side, level, gpuFormat, width, height, blockLength, bytes) {
+		reader.readTextures(function(side, level, gpuFormat, size, unused, blockLength, bytes) {
```

Comment:
Normally in Haxe we use `_` for "unused" stuff :)

Reply:
I was actually looking for that and I think I tried that and it failed 🤔 Does it maybe interfer with our global `_` from the game?